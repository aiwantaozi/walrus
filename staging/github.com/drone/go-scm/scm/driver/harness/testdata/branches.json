[
  {
    "name": "bla",
    "sha": "0c221fd126b9457d0ad2037641416083549f59c5",
    "commit": {
      "sha": "0c221fd126b9457d0ad2037641416083549f59c5",
      "title": "Create bla_file",
      "message": "Create bla_file",
      "author": {
        "identity": {
          "name": "thomas.honey",
          "email": "thomas.honey@harness.io"
        },
        "when": "2023-02-09T11:20:14Z"
      },
      "committer": {
        "identity": {
          "name": "Harness",
          "email": "noreply@harness.io"
        },
        "when": "2023-02-09T11:20:14Z"
      }
    }
  },
  {
    "name": "branch3",
    "sha": "59e1cdf0e421fd14b106f4861fb02dfd56b4dc34",
    "commit": {
      "sha": "59e1cdf0e421fd14b106f4861fb02dfd56b4dc34",
      "title": "delete README.2",
      "message": "delete README.2\n\ndelete README.2",
      "author": {
        "identity": {
          "name": "thomas.honey",
          "email": "thomas.honey@harness.io"
        },
        "when": "2023-02-09T12:39:37Z"
      },
      "committer": {
        "identity": {
          "name": "Harness",
          "email": "noreply@harness.io"
        },
        "when": "2023-02-09T12:39:37Z"
      }
    }
  },
  {
    "name": "main",
    "sha": "de2837f8911710cfb7bbb323d0de285fd2ef9155",
    "commit": {
      "sha": "de2837f8911710cfb7bbb323d0de285fd2ef9155",
      "title": "delete README.2",
      "message": "delete README.2\n\ndelete README.2",
      "author": {
        "identity": {
          "name": "thomas.honey",
          "email": "thomas.honey@harness.io"
        },
        "when": "2023-02-09T12:39:37Z"
      },
      "committer": {
        "identity": {
          "name": "Harness",
          "email": "noreply@harness.io"
        },
        "when": "2023-02-09T12:39:37Z"
      }
    }
  }
]