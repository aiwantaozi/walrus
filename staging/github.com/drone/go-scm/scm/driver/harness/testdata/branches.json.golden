[
    {
        "Name": "bla",
        "Path": "refs/heads/bla",
        "Sha": "0c221fd126b9457d0ad2037641416083549f59c5"
    },
    {
        "Name": "branch3",
        "Path": "refs/heads/branch3",
        "Sha": "59e1cdf0e421fd14b106f4861fb02dfd56b4dc34"
    },
    {
        "Name": "main",
        "Path": "refs/heads/main",
        "Sha": "1d640265d8bdd818175fa736f0fcbad2c9b716c9"
    }
]